src/client/datascience/notebook/contentProvider.ts

```diff
@@ -36,23 +36,16 @@ import { NotebookEditorCompatibilitySupport } from './notebookEditorCompatibilit
  */
 @injectable()
 export class NotebookContentProvider implements VSCNotebookContentProvider {
-    public webviews = new Map<string, NotebookCommunication[]>();
     constructor(
         @inject(INotebookStorageProvider) private readonly notebookStorage: INotebookStorageProvider,
         @inject(NotebookEditorCompatibilitySupport)
         private readonly compatibilitySupport: NotebookEditorCompatibilitySupport,
         @inject(IVSCodeNotebook) readonly notebookProvider: IVSCodeNotebook
-    ) {
-        notebookProvider.onDidCloseNotebookDocument(this.onDidCloseNotebook.bind(this));
-    }
-    public async resolveNotebook(document: NotebookDocument, webview: NotebookCommunication): Promise<void> {
-        // Add webviews to list. Used for testing
-        let list = this.webviews.get(document.uri.toString());
-        if (!list) {
-            list = [];
-            this.webviews.set(document.uri.toString(), list);
-        }
-        list.push(webview);
+    ) {}
+    public async resolveNotebook(_document: NotebookDocument, _webview: NotebookCommunication): Promise<void> {
+        // This function is due for deprecation. Associated code has been moved to
+        // NotebookKernelProvider will remove when removed from the API fully
+        return Promise.resolve();
```

Comment:
Better way to remove this? It's due for deprecation, but still in the proposed.d.ts. Didn't think that manually editing the proposed was a good idea, so I just moved out the code and made it a no-op.